api/v1alpha1/virtualmachinepublishrequest_types.go

```diff
@@ -82,6 +82,10 @@ type VirtualMachinePublishRequestSource struct {
 type VirtualMachinePublishRequestTargetItem struct {
 	// Name is the name of the published object.
 	//
+	// If the spec.target.location.apiVersion equals imageregistry.vmware.com/v1alpha1 and
```

Comment:
Please ensure the column length is <80 since all other comments in this file are.